tests/console/yast2_lan.pm

```diff
@@ -38,7 +39,8 @@ sub run {
     script_run('ls -alF /etc/sysconfig/network/');
     save_screenshot;
 
-    my $opened = open_yast2_lan();
+    my $y2_opts = is_pvm_hmc() ? "ncurses": "";
```

Comment:
same to the above, maybe it is better without "()"

Reply:
and this line needs tidying

Reply:
tidy fixed, thanks!